frontend/src/Pages/PrizesPage/tab.jsx

```diff
@@ -0,0 +1,87 @@
+import styled, { keyframes, css } from 'styled-components';
+import React, { useState } from 'react';
+
+const spin = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+const Gear = styled.div`
+  font-size: 32px;
+  // Removed margin-bottom as it's not needed anymore
+  animation: ${(props) =>
+    props.spinning
+      ? css`
+          ${spin} 2s linear infinite
+        `
+      : 'none'};
+  user-select: none;
+`;
+
+const TabContainer = styled.div`
+  display: flex;
+  z-index: 10;
+  position: relative;
+  text-align: ${(props) => (props.centerTitle ? 'center' : 'left')};
+  flex-direction: row;
+  align-items: center;
+  justify-content: ${(props) => (props.centerTitle ? 'center' : 'flex-start')}; /* Conditionally set justification */
+  cursor: pointer;
+  white-space: nowrap;
+  font-size: 20px;
+  user-select: none;
+  width: 250px; /* Added a fixed width for consistency */
+`;
+
+const Dropdown = styled.div`
+  background-color: #353535;
+	border-radius: 10px;
+  border: 1px solid #ccc;
+  z-index: 1;
+  width: 300px;
+  height: ${(props) => (props.open ? '225px' : '0')};
+  overflow-y: hidden;
+  transition: height 0.5s ease-in-out;
+  border: none;
+  display: block;
+  min-height: fit-content;
+`;
+
+const Prize = styled.div`
+  padding: 8px 16px;
+  color: white;
+  white-space: normal;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
```

Comment:
Can we make this a sans-serif font like the Poppins Font like use on other pages on the website?